app/helpers/finance/npq_payments_helper.rb

```diff
@@ -0,0 +1,157 @@
+# frozen_string_literal: true
+
+module Finance
+  module NPQPaymentsHelper
```

Comment:
Yeah I agree something like this would be much better to move to, but I guess the question is when to do it

Reply:
I am happy to split this PR into different parts say: 1) the fixes, 2) the deletions, 3) start a new branch to include something like this - but it will obviously need a bit more work

Reply:
I'd favour as separate class, business logic has nothing to do with the database. 
 I think let's not go too crazy with splitting this further now.